ui/StatusQ/src/StatusQ/Components/StatusSmartIdenticon.qml

```diff
@@ -44,7 +44,6 @@ Loader {
                            root.asset.bgColor
                 image.fillMode: root.asset.cropRect ? Image.PreserveAspectCrop
                                                     : Image.PreserveAspectFit
-                image.scale: root.asset.scale
```

Comment:
Looks suspicious, isn't scale used anymore?

Reply:
It's not used anywhere